samples/unit-tests/series-column/data-decimalpoint-negative/demo.js

```diff
@@ -0,0 +1,26 @@
+QUnit.test("Data module with decimapPoint and negative numbers (#4749)", function (assert) {
```

Comment:
The test is about the data module, so it should be located in `unit-tests/data/general`.